code/modules/supply/supply_pack.dm

```diff
@@ -0,0 +1,33 @@
+/datum/supply_packs
+	var/name
+	var/list/contains = list()
+	var/manifest = ""
+	var/amount
+	var/cost //default amount to cover crate cost?
+	var/containertype = /obj/structure/closet/crate
+	var/containername
+	var/access
+	var/hidden = FALSE
+	var/contraband = FALSE
+	var/group = SUPPLY_MISC
+	///Determines which departments do not need QM approval to order this supply pack
+	var/list/department_restrictions = list()
+
+	var/list/announce_beacons = list() // Particular beacons that we'll notify the relevant department when we reach
+
+	var/special = FALSE //Event/Station Goals/Admin enabled packs
+	var/special_enabled = FALSE
+
+	/// List of names for being done in TGUI
+	var/list/ui_manifest = list()
+
+/datum/supply_packs/New()
+	manifest += "<ul>"
```

Comment:
This one doesn't call its parents. Can be fine in certain cases...

Reply:
added parent call in next commit